main.go

```diff
@@ -245,7 +256,11 @@ func main() {
 	jsonBindF := kingpin.Flag("json-bind", "JSON bind address and port").Default("127.0.0.1:9922").String()
 	debugBindF := kingpin.Flag("listen-debug-addr", "Debug server listen address").Default("127.0.0.1:9933").String()
 	dataRetentionF := kingpin.Flag("data-retention", "QAN data Retention (in days)").Default("30").Uint()
-	dsnF := kingpin.Flag("dsn", "ClickHouse database DSN").Default("clickhouse://127.0.0.1:9000?database=pmm&block_size=10000&pool_size=2").String()
+	dsnF := kingpin.Flag("dsn", "ClickHouse database DSN. Can be override with database/host/port options").Default(defaultDsnF).String()
+	clickHouseDatabaseF := kingpin.Flag("clickhouse-name", "Clickhouse database name").Default("pmm").Envar("PMM_CLICKHOUSE_DATABASE").String()
+	clickhouseHostF := kingpin.Flag("clickhouse-host", "Clickhouse database hostname").Default("127.0.0.1").Envar("PMM_CLICKHOUSE_HOST").String()
+	clickhousePortF := kingpin.Flag("clickhouse-port", "Clickhouse database port").Default("9000").Envar("PMM_CLICKHOUSE_PORT").String()
```

Comment:
I suggest to use one env variable instead of 3. Any reason to use 3?

Reply:
https://github.com/percona/pmm-update/pull/172/files#diff-f5c4a5eeb44932a32574d75f25a0e21f147316c2e5790ced701489286918c690R30
We need to have separate parameters for database/port because we use it in for provisioning datasources in Grafana.
I think we have three options:
1. We can have three parameters (hostname, port, database name). Current solution.
2. We can merge host and port and have two variables. (host + port, database name)
3. Or we can have only PMM_CLICKHOUSE_URL and parse it on pmm-managed stage. It'll be sligly less obvious but we add note to datasources.yml.

Reply:
options 2 and 3 looks good to me. I'm not sure that we need database name. What about auth credentials?

Reply:
@BupycHuk We don't have auth creds/TLS for Victoria Metrics and Clickhouse yet and I think it'll be better to do in a separate task.